### PR TITLE
10.13.0 German translation User Guide

### PR DESCRIPTION
**Pull request by MWindel**
Added German translations for 10.13.0 to user guide

### CODE REVIEW

File: content/benutzerhandbuch/administration-de-bundle/audit-logs.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 20
 
 Audit-Logs zeigen die von Benutzern ausgeführten Operationen.
 
-Um die Audit-Logs-Liste anzuzeigen, klicken Sie auf **Audit Logs** im Menü **Konto**. Für jeden Logeintrag werden die folgenden Informationen bereitgestellt:
+Um die Audit-Logs-Liste anzuzeigen, klicken Sie auf **Audit-Logs** im Menü **Konten**. Für jeden Logeintrag werden die folgenden Informationen bereitgestellt:
```
Comment (MWindel) [inline content/benutzerhandbuch/administration-de-bundle/audit-logs.md:8]: check in UI

File: content/benutzerhandbuch/administration-de-bundle/audit-logs.md [hunk 2]
```diff
@@ -26,13 +26,13 @@ Um die Audit-Logs-Liste anzuzeigen, klicken Sie auf **Audit Logs** im Menü **Ko
 </tr>
 
 <tr>
-<td align="left">Ändern</td>
-<td align="left">Typ der Operationen, z. B. "Alarm erstellt", "Smart Rule gelöscht". Darunter wird der Benutzer angezeigt, der die Operation verarbeitet hat.</td>
+<td align="left">Ereignis</td>
```
Comment (MWindel) [inline content/benutzerhandbuch/administration-de-bundle/audit-logs.md:29]: Check in Strings/UI

**Review by rssag — APPROVED** [review]

File: content/benutzerhandbuch/cockpit-de-bundle/dashboards.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ weight: 40
 
 Dashboards ermöglichen eine individuelle Visualisierung Ihrer Daten anhand verschiedener Widgets. Widgets können Karten, Bilder, Graphen, Tabellen und andere grafische Darstellungen von Daten anzeigen.
 
-{{< product-c8y-iot >}} bietet eine Reihe voreingestellter Widgets, Details finden Sie im Abschnitt [Widgets-Sammlung](#widgets-collection). Sie können auch eigene Widgets entwickeln und Ihrem {{< product-c8y-iot >}}-Konto hinzufügen. Weitere Informationen finden Sie im [Web SDK Guide](/web/).
+{{< product-c8y-iot >}} bietet eine Reihe voreingestellter Widgets, Details finden Sie im Abschnitt [Widgets-Sammlung](#widgets-collection). Sie können auch eigene Widgets entwickeln und Ihrem {{< product-c8y-iot >}}-Konto hinzufügen. Weitere Informationen finden Sie im [Web SDK guide](/web/).
```
Comment (BeateRixen) [inline content/benutzerhandbuch/cockpit-de-bundle/dashboards.md:8]: Im Deutschen schreiben wir "guide" immer groß.

**Review by BeateRixen — APPROVED** [review]

File: content/benutzerhandbuch/cockpit-de-bundle/dashboards.md (hunk 3 above)
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/dashboards.md:8]:
```suggestion
{{< product-c8y-iot >}} bietet eine Reihe voreingestellter Widgets, Details finden Sie im Abschnitt [Widgets-Sammlung](#widgets-collection). Sie können auch eigene Widgets entwickeln und Ihrem {{< product-c8y-iot >}}-Konto hinzufügen. Weitere Informationen finden Sie im [Web SDK Guide](/web/).
```

**Pull request merged**